README.md

```diff
@@ -71,7 +87,15 @@ For more information, please see the [WFC3 Data Handbook](https://hst-docs.stsci
 export oref="/grp/crds/cache/references/hst/"
 ```
 
-For more information, please see the [STIS Data Handbook](https://hst-docs.stsci.edu/stisdhb). 
+For more information, please see the [STIS Data Handbook](https://hst-docs.stsci.edu/stisdhb)
+and the [stistools documentation](https://stistools.readthedocs.io/).
+
+## Other Software
+
+The following are also part of HST pipeline but not in this package:
```

Comment:
Change: "following are also part of HST pipeline but not in this package" to "following are also part of the standard HST pipeline, but are independent software packages written in Python or Python layered on C: calcos, costools, and drizzlepac.